### ISSUE TITLE
Screen reading software cannot navigate to completion focus

### ISSUE THREAD

**Issue by qq3477906169** [HabitRPG/habitica-android#1540]
Hello developer, I am a visually impaired user, I cannot complete Dailies because the screen reader cannot navigate to the completed focus, I hope to solve it, thank you very much
Device: HUAWEI GLK-AL00
Android Version: 29
AppVersion: 版本3.2.3 (2934)
User ID: 88079191-691e-4d5f-9ed5-b7b12611e3ab
Level: 5
Class: rogue
Is in Inn: false
Uses Costume: true
Custom Day Start: 0
Timezone Offset: -480
Details:

**Comment by Trion129** [comment]
I'll take a look.

**Comment by Trion129** [comment]
Tried adding contentDescription to the button. Then installed Talkback app to see the experience. Not able to focus on the specific checkbox yet. Will try to fix it. While at it I also found that Todo has similar problem. Will try to fix it too.

**Comment by Trion129** [comment]
Got it to focus on the checkbox and speak it out. But I found another bug with this. So, when I use talkback to click any of the buttons, it doesn't update the state. This is for all buttons. So, let's say I press + in Habits, it doesn't update the streak count. A checkbox in dailies doesn't mark the object as completed. Will try to fix this too, interesting to learn about accessibility through this issue.
